### ISSUE TITLE
pmempool: sync: improper uuids update

### ISSUE THREAD

**Issue by kamdiedrich** [pmem/issues#314]
Consider following local pool set file1:

```
PMEMPOOLSET
8M /tmp/part.0
REPLICA
8M /tmp/replica.1
```

and following local pool set file2:

```
PMEMPOOLSET
8M /tmp/part.2
```

Call pmempool create obj on above files.
Create following local pool set file3:

```
PMEMPOOLSET
8M /tmp/part.0
REPLICA
8M /tmp/replica.1
REPLICA
8M /tmp/part.2
```

Spoil replicas from pool set file1 by edditing file, write some incorrect data.
Then call pmempool sync on pool set file3. Exit code is 0, replicas will be synchronized but uuids where not updated.

Found on: Found on: 1.1-879-g3d09c5c

**Comment by plebioda** [comment]
Looks like this issue has been fixed in https://github.com/pmem/nvml/pull/1359. @kamdiedrich Could you please verify this ?

**Comment by kamdiedrich** [comment]
I was able to reproduce it on version: 1.1-1042-g260c6b2

**Comment by wojtuss** [comment]
The issue was fixed in 1.1-1044-gded3fbe

**Comment by kamdiedrich** [comment]
Verified on: 1.1-1092-gdbefada
